packages/blade/src/components/Icons/_Svg/Circle/__tests__/Circle.native.test.tsx

```diff
@@ -5,7 +5,7 @@ import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
 describe('<Circle />', () => {
   it('should render react-native-svg Circle component', () => {
     const renderTree = renderWithTheme(
-      <Svg width="400" height="110" viewBox="0 0 100 100">
+      <Svg width="400px" height="110px" viewBox="0 0 100 100">
```

Comment:
Can you check if this works on react native?

The underlying lib that we use has a usage guideline but couldn't see any `px` values being accepted there. https://github.com/software-mansion/react-native-svg/blob/main/USAGE.md

Reply:
Seems to work

Reply:
Can't we keep using the unitless values as `string`? Would translate to px values 🤔

Reply:
+1 to this. I'm still a bit skeptical about adding px here because of the RN library. Can we keep it as it was?

Reply:
Reverted this change. I was confusing the height width of SVG with the height width of CSS and in Box CSS we had decided to keep values with units. They are different so we don't have to change things here.

Also tried out SVGs in RN and they seem to work